real_time/arduino_src/radio_buggy_mega/main.cpp

```diff
@@ -360,7 +417,6 @@ void brake_init()
 void brake_raise() 
 {
     BRAKE_OUT_PORT |= _BV(BRAKE_OUT_PINN);
-    BRAKE_INDICATOR_PORT &= ~_BV(BRAKE_INDICATOR_PINN);
```

Comment:
At the moment the hardware doesn't support that since we only have 3 LEDs and only wanted to indicate faults/errors.  Given that there are 5 ways the brakes could be dropped (Watchdog timeout, RC timeout, Auton timeout, RC commanded, and Auton commanded), only 4 of which are currently supported, one could infer using our status lights why the brakes have dropped.